### PR TITLE
VS-974 - Better error on GvsCreateFilterSet.PopulateFilterSetInfo

### PR DESCRIPTION
**Pull request by gbggrant**
Changed so that PopulateFilterSetInfo now explicitly prints to STDERR any error it encounters. 

Example run [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/2f86bfa0-bf1b-451a-8217-a0ab4cba5834).
stderr has this error now:

> Error loading combined TSV into gvs-internal.gg_VS_1056.filter_set_info:
> BigQuery error in load operation: Provided Schema does not match Table gvs-
> internal:gg_VS_1056.filter_set_info. Cannot add fields (field: score2)

### CODE REVIEW

File: scripts/variantstore/wdl/GvsAssignIds.wdl [hunk 1]
```diff
@@ -15,7 +15,6 @@ workflow GvsAssignIds {
     File external_sample_names
     Boolean samples_are_controls = false
 
-    File? assign_ids_gatk_override
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsAssignIds.wdl:18]: unused variable.

**Review by rsasch — APPROVED** [review]
LGTM but an even simpler solution would just be to set the option to fail if an error is returned, don't send the error into a file (it will go into the logs), and don't return the file as a task output

File: scripts/variantstore/wdl/GvsCreateFilterSet.wdl [hunk 2]
```diff
@@ -43,7 +43,7 @@ workflow GvsCreateFilterSet {
   String fq_filter_sites_destination_table =    "~{project_id}.~{dataset_name}.filter_set_sites"
   String fq_filter_set_info_destination_table = "~{project_id}.~{dataset_name}.filter_set_info"
 
-  String filter_set_info_destination_table_schema = "filter_set_name:string,type:string,location:integer,ref:string,alt:string,calibration_sensitivity:float,score:float,vqslod:float,culprit:string,training_label:string,yng_status:string"
+  String filter_set_info_destination_table_schema = "filter_set_name:string,type:string,location:integer,ref:string,alt:string,calibration_sensitivity:float,score:float,score2:float,vqslod:float,culprit:string,training_label:string,yng_status:string"
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsCreateFilterSet.wdl:46]: what's this change for?

Comment (gbggrant) [reply scripts/variantstore/wdl/GvsCreateFilterSet.wdl:46]: That's just there for testing. Gone in 10 seconds.

**Pull request merged**